<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-MNK2190" failures="2" tests="5" name="TestNG test-3" time="0.074" errors="0" timestamp="2021-07-26T17:17:18 IST">
  <testcase classname="TestCase.Test_dependency" name="name1" time="0.002">
    <failure type="java.lang.AssertionError" message="name-1 fail">
      <![CDATA[java.lang.AssertionError: name-1 fail
at TestCase.Test_dependency.name1(Test_dependency.java:11)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- name1 -->
  <testcase classname="TestCase.Test_dependency" name="name3" time="0.001">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at TestCase.Test_dependency.name3(Test_dependency.java:22)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- name3 -->
  <testcase classname="TestCase.Test_dependency" name="name4" time="0.002"/>
  <testcase classname="TestCase.Test_dependency" name="name5" time="0.002"/>
  <testcase classname="TestCase.Test_dependency" name="name2" time="0.001">
    <skipped/>
  </testcase> <!-- name2 -->
</testsuite> <!-- TestNG test-3 -->
